index.bs

```diff
@@ -1447,16 +1459,17 @@ The following operations can be invoked by the client in an authenticator sessio
 This operation must be invoked in an authenticator session which has no other operations in progress. It takes the following
 input parameters:
 
-- The caller's RP ID, as determined by the user agent and the client.
+- The caller's [=RP ID=], as optionally decared by the [=[RP]=] and vetted by the user agent and the client, or defaulted to the
```

Comment:
nit- I almost wonder at the language here, since the definition of RP ID includes how it's vetted... but I think that's more style than substance, so ... proceed!